src/main.rs

```diff
@@ -47,13 +48,20 @@ impl Buildpack for DebianPackagesBuildpack {
     type Error = DebianPackagesBuildpackError;
 
     fn detect(&self, context: DetectContext<Self>) -> libcnb::Result<DetectResult, Self::Error> {
-        if BuildpackConfig::exists(context.app_dir.join("project.toml"))? {
+        let log = Print::new(stdout()).without_header();
+        if let Some(project_toml) = get_project_toml(&context.app_dir)? {
+            if BuildpackConfig::is_present(project_toml)? {
+                DetectResultBuilder::pass().build()
+            } else {
+                log.important("Project.toml found, but no [com.heroku.buildpacks.deb-packages] configuration present.").done();
```

Comment:
@colincasey I've just noticed this in a local build - `Project.toml` has the wrong capitalisation - could you change to `project.toml`? :-)

Reply:
I've opened #97 for this